ci/do_ci.sh

```diff
@@ -29,7 +29,7 @@ function test_salvo() {
   pushd salvo
 
   install_deps
-  bazel test //...
+  tools/coverage.sh
```

Comment:
Not sure what the implications for Python are, but in other languages `blaze coverage` builds in a different mode than `blaze test`, so we usually keep both modes.

This might be moot point for Python, so if you believe `blaze coverage` gives us everything we have with `blaze test`, it is probably fine.

Reply:
Good question.  I was under the impression that the test modes would be congruent so running the tests under the coverage script would work the same way. as the plain "test" target.

I don't think the duplication by running "bazel test" in addition to the coverage buys us much.   I'll do some more research and hold the option to update this later.  Would that work?

Reply:
That sounds good, thank you @abaptiste.